### DISCUSSION TITLE
Can I save applied filters to Database. So that they can be used as filter templates later.

### DISCUSSION THREAD

**Discussion by vin33t** [Power-Components/livewire-powergrid#1401] [Q&A]
Persisted save only save it to cookies and can not be saved permanently.
Looking for a way to render the table with filters applied from saved filter templates

**Comment by whjvdijk** [comment]
Want the same thing. I need users to be able to create custom filters combinations (site, country, productgroup etc.) and then save that for easy access to the same datagroup later.
